### PR TITLE
Actually use the 'defaultValue' property in AlertViewIOS.

### PR DESCRIPTION
**Pull request by mpretty-homepass**
The AlertViewIOS component takes in a 'defaultValue' for the text input but never actually sets it, this PR actually sets the value.

### CODE REVIEW

**Comment by grabbou** [comment]
Hey, looks really cool! Would you mind providing a small JS snippet & native screenshot to provide it works as desired?

File: React/Modules/RCTAlertManager.m [hunk 1]
```diff
@@ -168,25 +168,29 @@ - (void)invalidate
                                           message:nil
                                    preferredStyle:UIAlertControllerStyleAlert];
     switch (type) {
-      case UIAlertViewStylePlainTextInput:
+      case UIAlertViewStylePlainTextInput: {
```
Comment (nicklockwood) [inline React/Modules/RCTAlertManager.m]: Why add brackets?

Comment (mpretty-homepass) [inline React/Modules/RCTAlertManager.m]: Wasn't compiling without them.

Comment (mpretty-homepass) [inline React/Modules/RCTAlertManager.m]: Feel free to remove them if it compiles for you.

**Comment by mpretty-homepass** [comment]
@grabbou The UIExplorer project actually already has an example which should have a 'Default Value' - https://github.com/facebook/react-native/blob/master/Examples/UIExplorer/AlertIOSExample.js#L141

**Comment by mpretty-homepass** [comment]
@nicklockwood Just merged to latest master because there was an issue in the UIExplorer project unrelated to my change.

**Comment by nicklockwood** [comment]
@facebook-github-bot shipit

**Comment by mpretty-homepass** [comment]
@nicklockwood just FYI, I'm pretty sure this commit https://github.com/facebook/react-native/commit/6d0272af424fc79e60789a8885460f581f1d1749 will break the pull request (the 'unused variable' is the variable I'm using in the PR).

Won't be able to update this until next week though :/

**Pull request closed without merging**